ts/credentials.ts

```diff
@@ -15,7 +15,7 @@ creds.getPromise().then(
 );
 
 if (creds.needsRefresh()) {
-    creds.refresh(err => {
+    creds.refresh((err = undefined) => {
```

Comment:
Looks like defaul value [was added in Node6](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters), since the library still needs to be run on older node versions(>= 0.10), we cannot use this synctax for now.

Reply:
This is the typescript typing. Wouldn't TSC take care of the target node version transpilation under this constraint? I am assuming those settings to be available in tsconfig as we are already supporting older versions.

Reply:
@AllanFly120 does the above address/satisfy your concern? The reckon this is a simple PR and would be nice to act on it sooner. :)